test/tarantool_disconnect.c

```diff
@@ -14,7 +14,8 @@ int main() {
 	tnt_set(tnt, TNT_OPT_RECV_BUF, 0); // Disable buffering for recv
 	tnt_connect(tnt);
 
-	while (1) {
+	for (int i = 0; i < 10; i++)
+	{
```

Comment:
What is the point of the change? AFAIU it is the test is for manual testing: you run the server, run the client and perform several stop / start of the server. The client should reconnect and process responses correctly (see #41).

I reverted this hunk, built the connector and the test with ASAN and after Ctrl+C it does not show anything bad on the terminal.

Reply:
Ok, i revert this part.